ramls/request.json

```diff
@@ -231,6 +231,10 @@
       "description": "Request fields used for search",
       "type": "object",
       "$ref": "request-search-index.json"
+    },
+    "itemLocationCode": {
```

Comment:
Can we have a test which checks that a request can actually be created with this new property?

Reply:
Added test